### ISSUE TITLE
And I press "Toggle" Behat Step failing in Tiny_wiris

### ISSUE THREAD

**Issue by TheBryanOpen228** [wiris/moodle-filter_wiris#143]
While adding Tiny_wiris i found a problem with the Behat automated tests from _lib/editor/tiny/plugins/wiris/tests/behat_
Specially in the step 
`And I press "Toggle" in "FIELDNAME" field in TinyMCE 6 editor`

For example in edit_formula.feature, see:
```
Scenario: Create a formulas                                               # /builds/moodle/lib/editor/tiny/plugins/wiris/tests/behat/edit_formula.feature:17
And I press "Toggle" in "General description" field in TinyMCE 6 editor  # /builds/moodle/lib/editor/tiny/plugins/wiris/tests/behat/edit_formula.feature:24
"Toggle" button not found in "General description" field
```

This caused by a string change here in filter_wiris, see [fix: Test for Moodle 4.4 compatibility](https://github.com/wiris/moodle-filter_wiris/commit/fad00f0ae28ebba4b41989ae4723778a9307cb9b#diff-510a3b5e3c4056f806b1b066d41f883e21d8444f9d4a4aba3b06d1700ac56f27R258)
The string changed from "More..." to "Reveal or hide additional toolbar items", but i don't see it added in any language File, so the Toggle button it's actually showing "More..." text.

So changing "Reveal or hide additional toolbar items" back to "More..." fix the issue, here in [behat_wiris_page.php](https://github.com/wiris/moodle-filter_wiris/blob/fad00f0ae28ebba4b41989ae4723778a9307cb9b/tests/behat/behat_wiris_page.php#L258)
`"Toggle" => "More..."`

Can you fix those Behat steps? Thanks in advice

**Comment by carla-at-wiris** [comment]
Hello @TheBryanOpen228,

Moodle changed the title assigned to the Toggle button on their latest version `4.4`, so we had to change our tests to be compatible with it. We realized that for prior versions there would be incompatibilities, so we tried to fix it on this [Pull request](https://github.com/wiris/moodle-filter_wiris/pull/141/files#diff-510a3b5e3c4056f806b1b066d41f883e21d8444f9d4a4aba3b06d1700ac56f27). It is still on review since we are making big changes on our tests suites.

We will investigate the behavior of our tests when different languages are set on the Moodle instance. However, knowing the Moodle version you are using will also help us a lot!

We will let you know once we release the previous mentioned changes and any new regarding the matter.

**Comment by TheBryanOpen228** [comment]
Hello, Thanks for the information.
We are using 8.7.0 (2024042300) version for tiny_wiris and filter_wiris. 
Also, we use Moodle 4.3.4 (Build: 20240422)

So if that's the case, it explains why in 4.3 i see the error, since this change is for 4.4 Moodle Version. Then the fix is probably not needed.

**Comment by usantos-at-wiris** [comment]
We just released a new moodle version 8.7.1 fixing this issue. Thanks a lot for reporting it !
